src/Documentation/UserManual/DeviceOpticalMarkerTracker.dox

```diff
@@ -0,0 +1,86 @@
+
+- Print the <a href="https://www.assembla.com/spaces/plus/documents/cd0kq6t7ir577cdmr6QqzO/download/cd0kq6t7ir577cdmr6QqzO">aruco calibration chessboard</a> and measure the side length of the markers in meters.
+- To generate a calibration file, run aruco_calibration.exe using the command
+  \code aruco_calibration.exe live:<camera-index> <output-file-name>.yml -size <marker-size-m> \endcode
+  where:
+  - <camera-index> is the integer index of the camera you want to use (usually 0 or 1).
+  - <output-file-name> is the desired output file name.
+  - <marker-size-mm> is the size of a marker side in meters.
+- With the video window selected:
+  - Press \b a to capture image and add to the calibration image set.
+  - Press \b c to run calibration on the captured images.  "repj" is a measure of the calibration error (normal is < 1).
+  - Press \b s to stop calibration.
+
+- For best results use at least 6 images including:
+  - At least one frontal image. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Frontal.png
+  - Multiple images where the border of the chessboard is near the top or side image border. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Edge.png
+  - Multiple images where the chessboard is held at an oblique angle to the camera. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Oblique1.png
+    \image html ../images/DeviceOpticalMarkerTracker/Oblique2.png
+
+\section DeviceOpticalMarkerTrackerImages Creating images for printing
+- To print a \b single \b image, run aruco_print_marker.exe using the command
+  \code aruco_print_marker.exe <marker-id> <output-file-name>(.jpg|.png|.ppm|.bmp) -bs <bit-size> -d <dictionary> \endcode
+  where:
+  - <marker-id> is the integer ID of the marker you wish to print.
+  - <output-file-name> is the desired marker image filename.
+  - <bit-size> is the number of pixels per interior image bit \OptionalAtt{50px}.
+  - <dictionary> is the marker dictionary you wish to use (we suggest ARUCO_MIP_36h12).
+- To print an \b entire \b dictionary at once, run aruco_print_dictionary.exe using the command
+  \code aruco_print_dictionary.exe <output-directory> <dictionary> -s <bit-size> \endcode
+  where:
+  - <output-directory> is the directory to output the marker images to (must already exist).
+  - <dictionary> is the marker dictionary you wish to use (we suggest ARUCO_MIP_36h12).
+  - <bit-size> is the number of pixels per interior image bit \OptionalAtt{75px}.
+
+\section DeviceOpticalMarkerTrackerConfigSettings Device configuration settings
+
+- \xmlAtt \ref DeviceType "Type" = \c "OpticalMarkerTracker" \RequiredAtt
+- \xmlAtt \b CameraCalibrationFile   Relative to \ref FileApplicationConfiguration "DeviceSetConfigurationDirectory". \RequiredAtt
+- \xmlAtt \ref ToolReferenceFrame \OptionalAtt{Tracker}
+- \xmlAtt \b TrackingMethod Tracking method. \OptionalAtt{OPTICAL_ONLY}
+    - \c OPTICAL_ONLY uses just RGB video.
+    - \c OPTICAL_DEPTH uses depth data and RGB video.
```

Comment:
Rename to OPTICAL_AND_DEPTH (to be consistent with other configuration options, such as replay from file: IMAGE, TRANSFORM, or IMAGE_AND_TRANSFORM).